python/ray/tests/test_gcs_fault_tolerance.py

```diff
@@ -47,7 +47,6 @@ def test_gcs_server_restart(ray_start_regular):
     assert result == 2
 
 
-@pytest.mark.skipif(new_scheduler_enabled(), reason="broken")
```

Comment:
@richardliaw  @wumuzi520 you two just disabled these yesterday. Can you verify that we can turn them back on?

Reply:
We turned them off due to Travis failing iirc.

Reply:
If travis doesnt fail anymore, then fine to revert.